tests/unit/api/v1/test_records.py

```diff
@@ -796,7 +797,7 @@ async def test_create_record_response_with_extra_question_responses(
                         "label_selection_question_1": {"value": False},
                     },
                 },
-                "False is not a valid option.\nValid options are: ['option1', 'option2', 'option3']",
+                "0 is not a valid option.\nValid options are: ['option1', 'option2', 'option3']",
```

Comment:
@frascuchon this is the same issue, `False` is consumed by Pydantic casting it to `int` (as `0` int).

Reply:
Super weird :-)

Reply:
Indeed.